runtime/compiler/env/VMJ9Server.cpp

```diff
@@ -2242,6 +2242,14 @@ TR_J9ServerVM::vTableOrITableIndexFromMemberName(TR::Compilation* comp, TR::Know
       }
    return -1;
    }
+
+TR::KnownObjectTable::Index
+TR_J9ServerVM::delegatingMethodHandleTarget( TR::Compilation *comp, TR::KnownObjectTable::Index dmhIndex, TR_OpaqueClassBlock *cwClass, TR::KnownObjectTable *knot)
+   {
+   JITServer::ServerStream *stream = _compInfoPT->getMethodBeingCompiled()->_stream;
+   stream->write(JITServer::MessageType::VM_delegatingMethodHandleTarget, dmhIndex, cwClass, knot);
```

Comment:
The client and the server have their own known object tables. Passing the one from server to the client will not work. In `handleServerMessage` we already have:
`TR::KnownObjectTable *knot = comp->getOrCreateKnownObjectTable();`